dev_docs/kibana_platform_plugin_intro.mdx

```diff
@@ -19,28 +19,43 @@ developer tools. The Kibana platform is a blank canvas, just waiting for a devel
 
 ![Kibana personas](assets/kibana_platform_plugin_end_user.png)
 
-## Platform services
+## 1,000 foot view
 
-Plugins have access to three kinds of public services:
+At a super high-level, Kibana is composed of **plugins**, **core**, and **Kibana packages**.
 
-- Platform services provided by `core` (<DocLink id="kibPlatformIntro" section="core-services" text="Core services"/>)
-- Platform services provided by plugins (<DocLink id="kibPlatformIntro" section="platform-plugins" text="Platform plugins"/>)
-- Shared services provided by plugins, that are only relevant for only a few, specific plugins (e.g. "presentation utils").
+![Kibana 1000 ft arch](assets/1000_ft_arch.png)
 
-The first two items are what make up "Platform services".
+**Plugins** provide the majority of all functionality in Kibana. All applications and UIs are defined here.
 
-<DocCallOut title="What is the difference between services provided by platform plugins, and those by core?">
+**Core** provides the runtime and the most fundamental services.
 
-We try to put only the most stable and fundamental code into `Core`, while more application focused functionality goes in a plugin, but the heuristic isn't
-clear, and we haven't done a great job of sticking to it. For example, notifications and toasts are core services, but data and search are plugin services.
+**@kbn packages** provide static utilities that can be imported anywhere in Kibana.
+
+<DocAccordion buttonContent="(Internal only) FAQ: Should I put my code in a plugin or a package?" color="warning">
+<DocCallOut color="warning">
+If it's stateful, it has to go in a plugin, but with stateless utilities, It Depends. Stateless code exported publicly from a plugin will increase the bundle size, however, the packages folder is far away from the plugins folder so having a part of your code in a plugin and the rest in a package may make it hard to find, leading to duplication.
```

Comment:
> Stateless code exported publicly from a plugin will increase the bundle size

Is this different than if the static code was exported from a package?

Reply:
Updated:

> Stateless code exported publicly from a plugin will increase the page load bundle size of _every single page_, even if none of those plugin's services are actually needed. With packages, however, only code that is needed for the current page is downloaded.